e2e-tests/web/browserstack/wdio-browserstack-chrome.conf.js

```diff
@@ -0,0 +1,7 @@
+const defaultConfig = require('./wdio-browserstack.conf')
```

Comment:
Shouldn't we type these files as well?

Reply:
Flow should not be added to this project as we have TypeScript. Also writing this in TS is not really possible: https://github.com/rah-foundation/website/pull/10

I think we should not hack our way around because in the end there are probably no types available for the configs.

Reply:
@Taggotty did you try ES6 syntax and modules though?

Reply:
See [comment](https://github.com/Integreat/integreat-app/pull/169#issuecomment-789753282)